service/src/main/java/uk/gov/hmcts/reform/fpl/validation/interfaces/HasEnteredEPOExcluded.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.fpl.validation.interfaces;
```

Comment:
minor: it would be nice to move these and the validators into their own epo package in here
i.e. `package uk.gov.hmcts.reform.fpl.validation.interfaces.epo;`